components/select/index.tsx

```diff
@@ -207,10 +207,16 @@ export default class Select<T = SelectValue> extends React.Component<SelectProps
       clearIcon,
       menuItemSelectedIcon,
       showArrow,
+      disableAnimation: disableAnimationLocal,
       ...restProps
```

Comment:
Ostensibly, you'd be able to extract `transitionName` and `choiceTransitionName` here, to use it for :ice_cream: